gnovm/stdlibs/std/frame.gno

```diff
@@ -0,0 +1,10 @@
+package std
+
+type Realm struct {
+	Addr    Address
```

Comment:
how about unexposed with getters? more future proof...

Reply:
Why not, Also for pkgPath ?

cc: @moul @thehowl wdyt ?

Reply:
Indeed, these system types inspire trust, even when passed or stored elsewhere.